Source/Diagnostics/WarpXOpenPMD.cpp

```diff
@@ -734,11 +769,12 @@ WarpXOpenPMDPlot::SetupRealProperties (openPMD::ParticleSpecies& currSpecies,
                       const amrex::Vector<std::string>& real_comp_names,
                       const amrex::Vector<int>& write_int_comp,
                       const amrex::Vector<std::string>& int_comp_names,
-                      unsigned long long np) const
+                      const unsigned long long &np, bool isBTD) const
```

Comment:
Fundamental types are ideally not passed by reference. `isBTD` can be `const` :)
```suggestion
                      const unsigned long long np, bool const isBTD) const
```